state-of-the-docs-2015.rst

```diff
@@ -0,0 +1,197 @@
+    requests, and be accountable for keeping everything running.
+
+    Currently, all money coming into Read the Docs is treated as income into the
+    business. We do track income to the community site separately, so we can
+    budget for the community site separately from running the rest of the
+    business.
+
+    We aren't yet to the point where this split is critical. There are no
+    employees of the business yet, and we are only able to pay ourselves for a
+    small portion of the work we perform for the community site.
+
+    We will need to track this explicitly as we grow however, as the community
+    site will need to rely on donations for the foreseeable future. Those
+    donations should not go towards developing the for-profit side directly.
+    Development of the community site will still be a primary function of income
+    into the for-profit side.
+
+    Eventually, the community site could be structured completely separate, to
+    provide a more explicit split.  It could probably be structured as a 501c,
+    This doesn't make financial sense yet.
+
+Our support fundraiser
+    In an effort to fund development and maintenance on the community site, we
+    decided to call on our community for financial support back in June.  This
+    was the first time we have asked the community for contributions.  Our goal
+    was to fund at least a portion of the time spent developing and supporting
+    the community site.
+
+    Thanks to all the contributors from our community, we raised our goal of
+    $24,000, and were able to fund the time of multiple engineers working
+    steadily on Read the Docs for the last few months.
+
+    We already outlined our costs and our plans behind our June fundraiser in a
+    previous post. We will be following up the last few months of work with an
+    overall review of the fundraiser and the work we performed in the coming
+    weeks.
+
+Triaging
+    Hopefully, our support responsiveness has been more apparent.  One hurdle in
+    bringing on more engineers, and engaging the community more, was the lack of
+    formal ticketing and open decision making.  Gregor, among many other
+    changes, laid out a formal triage process that has been a major influence in
+    keeping an organized and healthy support queue.
+
+    We've definitely found that the more support engagement we provide the
+    community, the more support comes out of the woodwork.  This is all part of
+    supporting a healthy community however, as we have learned.
+
+Stability
+    Read the Docs has "grown organically", which is to say it has grown fast,
+    unwieldy, and in many directions.  With more proper focus on code quality,
+    including clean up efforts, refactoring, and long due upgrades, Read the
+    Docs has shed considerable technical debt in the last couple months.  This
+    work has already paid off in improving stability and predictability of the
+    project.
+
+Gold
```

Comment:
This could do more to introduce the feature conceptually then, and still avoid promoting it as a usable feature. We haven't done anything to hide it thus far, I don't see mentioning it here working against promoting it later either.